src/og/scene/Planet.js

```diff
@@ -815,35 +815,65 @@ export class Planet extends RenderNode {
     }
 
     _preRender() {
+        // Zoom 0
         this._quadTree.createChildrenNodes();
-        this._quadTree.segment.createPlainSegment();
+        this._quadTree.segment.createPlainSegment();        
+
+        // Zoom 1
+        for (let i = 0; i < this._quadTree.nodes.length; i++) {
+            this._quadTree.nodes[i].segment.createPlainSegment();
+        }
+
+        // same for poles
+        this._quadTreeNorth.createChildrenNodes();
+        this._quadTreeNorth.segment.createPlainSegment();
+
+        for (let i = 0; i < this._quadTreeNorth.nodes.length; i++) {
+            this._quadTreeNorth.nodes[i].segment.createPlainSegment();
+        }
+
+        this._quadTreeSouth.createChildrenNodes();
+        this._quadTreeSouth.segment.createPlainSegment();
+
+        for (let i = 0; i < this._quadTreeSouth.nodes.length; i++) {
+            this._quadTreeSouth.nodes[i].segment.createPlainSegment();
+        }
+
+        this._preLoad();
+    }
+
+    _preLoad() {
```

Comment:
do we need add an option, for some scenario, we need preload, for some , we don't need?

Reply:
In most cases default this preLoad (for 0,1 zoom levels) is fine. If we meet a case when we do not need preLoad we will add the option.